src/main/java/me/zeroeightsix/kami/module/modules/client/Overlays.java

```diff
@@ -0,0 +1,93 @@
+        if (informationOverlayDisabled.getValue() != lastInformationOverlayDisabled) gui.setChildVisibilityByName("InformationOverlayFrame",informationOverlayDisabled.getValue());
+        if (inventoryViewerDisabled.getValue() != lastInventoryViewerDisabled) gui.setChildVisibilityByName("InventoryViewerFrame",inventoryViewerDisabled.getValue());
+        if (friendsListDisabled.getValue() != lastFriendsListDisabled) gui.setChildVisibilityByName("FriendsListFrame",friendsListDisabled.getValue());
+        if (textRadarDisabled.getValue() != lastTextRadarDisabled) gui.setChildVisibilityByName("TextRadarFrame",textRadarDisabled.getValue());
+        if (entitylistDisabled.getValue() != lastEntityListDisabled) gui.setChildVisibilityByName("EntityListFrame",entitylistDisabled.getValue());
+        if (coordinatesDisabled.getValue() != lastCoordinatesDisabled) gui.setChildVisibilityByName("CoordinatesFrame",coordinatesDisabled.getValue());
+
+        //reset the change checking variables to the new normal value
+        lastRadarDisabled = radarDisabled.getValue();
+        lastBaritoneDisabled = baritoneDisabled.getValue();
+        lastActiveModulesDisabled = activeModulesDisabled.getValue();
+        lastInformationOverlayDisabled = informationOverlayDisabled.getValue();
+        lastInventoryViewerDisabled = inventoryViewerDisabled.getValue();
+        lastFriendsListDisabled = friendsListDisabled.getValue();
+        lastTextRadarDisabled = textRadarDisabled.getValue();
+        lastEntityListDisabled = entitylistDisabled.getValue();
+        lastCoordinatesDisabled = coordinatesDisabled.getValue();
+    }
+
+    public void onDisable()
+    {
+        KamiGUI gui = KamiMod.getInstance().getGuiManager();
+        //When Overlay Manager is disabled, turn all of the overlays back on
+        gui.setChildVisibilityByName("RadarFrame",true);
+        gui.setChildVisibilityByName("BaritoneFrame",true);
+        gui.setChildVisibilityByName("ActiveModulesFrame",true);
+        gui.setChildVisibilityByName("InformationOverlayFrame",true);
+        gui.setChildVisibilityByName("InventoryViewerFrame",true);
+        gui.setChildVisibilityByName("FriendsListFrame",true);
+        gui.setChildVisibilityByName("TextRadarFrame",true);
+        gui.setChildVisibilityByName("EntityListFrame",true);
+        gui.setChildVisibilityByName("CoordinatesFrame",true);
+
+        //Change these variables so it will update when turned back on
+        lastRadarDisabled = true;
+        lastBaritoneDisabled = true;
+        lastActiveModulesDisabled = true;
+        lastInformationOverlayDisabled = true;
+        lastInventoryViewerDisabled = true;
+        lastFriendsListDisabled = true;
+        lastTextRadarDisabled = true;
+        lastEntityListDisabled = true;
+        lastCoordinatesDisabled = true;
+    }
```

Comment:
this shouldn't be changed on disable, instead overlay manager should always be on, like the other client modules